Tests/LibWeb/Text/input/iframe-removed-asyncronously.html

```diff
@@ -0,0 +1,20 @@
+<body>
+</body>
+<script src="include.js"></script>
+<script>
+    asyncTest(done => {
+        let frame = document.body.appendChild(document.createElement("iframe"));
+
+        setTimeout(() => {
+            frame.remove();
+
+            setTimeout(() => {
```

Comment:
This is a really weird test, but it seems to work fine...

Reply:
Sidenote - I don't understand the python script involvement here in the frame, took a bit to try and understand what was going on: https://github.com/web-platform-tests/wpt/blob/master/url/percent-encoding.window.js